### PR TITLE
Add downtime for UCSDT2 for work on cooling system

### PR DESCRIPTION
**Pull request by jdost321**

### CODE REVIEW

File: topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml [hunk 1]
```diff
@@ -729,3 +729,58 @@
...
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439220
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-SE
```
Comment (matyasselmeci) [inline topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: There's no "UCSDT2-SE" but there are "UCSDT2-SE1" and "UCSDT2-SE3" so this should be one of those.

Comment (jdost321) [reply topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: Sorry should have been UCSDT2-SE1

Comment (matyasselmeci) [reply topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: Can you push the change or should I make it?

Comment (jdost321) [reply topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: i just fixed it in a new commit on my branch, was that enough?

Comment (matyasselmeci) [reply topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml]: Yep, looks good.

**Pull request merged**